beacon-chain/rpc/eth/validator/handlers.go

```diff
@@ -157,3 +170,117 @@ func (s *Server) SubmitAggregateAndProofs(w http.ResponseWriter, r *http.Request
 		http2.HandleError(w, "Could not broadcast one or more signed aggregated attestations", http.StatusInternalServerError)
 	}
 }
+
+// SubmitSyncCommitteeSubscription subscribe to a number of sync committee subnets.
+//
+// Subscribing to sync committee subnets is an action performed by VC to enable
+// network participation, and only required if the VC has an active
+// validator in an active sync committee.
+func (s *Server) SubmitSyncCommitteeSubscription(w http.ResponseWriter, r *http.Request) {
+	if shared.IsSyncing(r.Context(), w, s.SyncChecker, s.HeadFetcher, s.TimeFetcher, s.OptimisticModeFetcher) {
```

Comment:
we should still have the trace added here right?

Reply:
I decided to remove them because they are not useful. I don't think we ever wanted to check the span of an endpoint. And as far as I know creating spans is expensive.